CMakeLists.txt

```diff
@@ -272,7 +272,7 @@ endif(WITH_LEVELDB)
 
 find_package(snappy REQUIRED)
 
-option(WITH_LZ4 "LZ4 compression support" OFF)
+option(WITH_LZ4 "LZ4 compression support" ON)
```

Comment:
```
CMake Error at /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find LZ4 (missing: LZ4_LIBRARY LZ4_INCLUDE_DIR) (Required is at
  least version "1.7")
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindLZ4.cmake:30 (find_package_handle_standard_args)
  CMakeLists.txt:277 (find_package)
```

again, i don't think we can enable LZ4 by default. for instance, this fails "run-make-check.sh" on trusty.